rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/ParticipantSex.java

```diff
@@ -0,0 +1,5 @@
+package pl.oaza.warszawa.dor.rekolekcje.api.participants.domain;
+
+public enum ParticipantSex {
+    FEMALE, MALE
+}
```

Comment:
Ok, Enum jest dobrym wyborem

Reply:
Wg mnie ten enum powinien być globalny, bo nie tylko uczestnik będzie miał płeć. Ale na razie możemy tak zostawić chyba. Jak dodamy więcej aktorów to wtedy to wydzielimy do jakiegoś współdzielonego modułu.